### DISCUSSION TITLE
Problem mit der Installation

### DISCUSSION THREAD

**Discussion by christianoblanco90** [camueller/SmartApplianceEnabler#330] [Installation und Betrieb]
Hallo zusammen, ich bin ein Fan meiner SMA-Anlage, aber die schlechte Konnektivität zu Geräte nervt mich. Daher schon mal vielen Dank für dieses Feature!

Aber leider komme ich bei der Standard-Installation nicht weiter. Ich habe die Image auf die Karte geladen und den USB-Stick ein und ausgesteckt, aber scheinbar scheint die 2. Phase der Installation nicht zu starten, denn nach dem Rausziehen der Sticks kommt ein reboot aber dann geht die Installation nicht weiter und irgendwann nach ein paar Sekunden steht da "raspberrypi login:" Aber eigentlich sollte ja nun ein Update etc. kommen.. :-(

Nutze einen Pi 3 Model B. Was mache ich nur falsch?

**Comment by Uwdmic** [comment]
Ahhh , der 3. mit mir wo die Installation vom USB Stick nicht startet

Les dir diesen Beitrag durch, so scheint es zu gehen
https://github.com/camueller/SmartApplianceEnabler/discussions/327

da mir auf meinen Post nicht geantwortet wurde, habe ich die Installation komplett manuell gemacht

**Reply by christianoblanco90** [reply]
Puh, also die manuelle Installation traue ich mir nicht zu. Wie würde man da am einfachsten vorgehen bzw. wie hast du das gemacht?

**Comment by camueller** [comment]
Auch wenn es 3 oder mehr Personen sind, bei denen die Installation nicht startet, heisst das nicht, dass die Installation grundsätzlich nicht funktioniert.

Ich habe vor wenigen Minuten erfolgreichreich eine [Installation entsprechend der Anleitung](https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/Installation_DE.md) durchgeführt. Dafür ist es allerdings notwendig, sich **exakt** an diese zu halten, was nicht jedem gelingt. Aber es gibt ja noch die Manuelle Installation ...

**Reply by christianoblanco90** [reply]
Was könnte denn ein Grund sein? Ich bin mir nicht sicher, wie ich genau die drei Text-Dateien auf den Stick bekommen soll. Habe einfach den Text (ohne Änderungen) in eine Text-Datei kopiert und auf den Stick gezogen. Aber leider kam bei drei verschiedenen Sticks nicht das gewünschte Ergebnis :-(

**Reply by camueller** [reply]
Den Text in eine neue Text-Datei kopieren ist halt nicht das gleiche, wie eine Datei einfach direkt auf den Stick herunterzuladen. Vor allem unter Windows kann das zu Problemen führen.

**Comment by camueller** [comment]
Für User, die bereits per SSH auf den Raspberry Pi kommen, werde ich spätestens morgen eine weitere Art der Installation dokumentieren, basierend auf https://github.com/camueller/SmartApplianceEnabler/discussions/327

**Reply by Uwdmic** [reply]
danke für deine Mühe, vielleicht könntest du, da auch dazuschreiben, wie man nach Installation des Betriebssystems SSH aktiviert, ich habe mich mit Tastatur und Monitor direkt am Raspy angemeldet, mit sudo raspi-config , kam ich direkt in ein grafisches Konfig-Menue, dort konnte ich ganz einfach SSH aktivieren

**Reply by camueller** [reply]
Das hat nichts mit dem SAE zu tun. Diese Informationen findet man in der Raspberry Pi Dokumentation.

**Reply by dennisbausch** [reply]
Wenn du das Image von hier nimmst hast du SSH bereits aktiviert , du kannst aber bei Image mit  Raspberry PI Imager SSH Aktivieren oder WLAN einrichten .

**Comment by camueller** [comment]
Es gibt jetzt eine weitere Art der Dokumentation:

https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/InstallationViaScript_DE.md
